src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -458,6 +458,13 @@ public function viewAction(int $orderId, Request $request): Response
         /** @var OrderSiblingProviderInterface $orderSiblingProvider */
         $orderSiblingProvider = $this->get('prestashop.adapter.order.order_sibling_provider');
 
+        $paginationNum = (int) $this->configuration->get('PS_ORDER_PRODUCTS_NB_PER_PAGE', 8);
+        $paginationNumOptions = [8, 20, 50, 100];
```

Comment:
Maybe this hardcoded array should be set as a constant? (it's possible to have a const containing an array as of PHP 5.6)

(idem for the value `8` in the line just above)

Reply:
Yep, good idea Done